bcrs-ui/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "hello-world",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "serve": "vue-cli-service serve",
+    "build": "vue-cli-service build",
+    "lint": "vue-cli-service lint",
+    "test:unit": "vue-cli-service test:unit --testPathPattern"
+  },
+  "dependencies": {
+    "@mdi/font": "^4.5.95",
+    "axios": "^0.18.0",
+    "core-js": "^3.4.3",
```

Comment:
I think you also need `"regenerator-runtime": "^0.13.3",`.
See comment in coops-ui, main.ts.

Reply:
k sounds good!